evm/vm/flavors/frontier/blocks.py

```diff
@@ -307,6 +306,9 @@ def add_uncle(self, uncle):
         self.header.uncles_hash = keccak(rlp.encode(self.uncles))
         return self
 
+    # TODO: Check with Piper what's the use case for having the mine() method allowing callsites
```

Comment:
I believe this functionality can be removed.  This was from an earlier architecture when I hadn't quite wrapped my head around what the block creation workflow was going to be.